deployment-bundle-steps.md

```diff
@@ -0,0 +1,135 @@
+# How to setup a project with deployment to S3
+
+1. Create S3 bucket for production / staging (your-bucket-name-production & your-bucket-name-staging). (If using cloudfront, see [optional settings](#optional-settings))
+
+2. Log into [circleci.com](https://circleci.com/) and add your project. Set up the following credentials: (see other members of the team for these values).
+  - Environment Variables
+    - GHUSER
+    - GHPASS
```

Comment:
I think we need standardisation regarding this across the business as in order to share things, we'd need accepted Environment Variable names. @hpoom this is one place where I do think we need a discussion.